modules/ROOT/pages/advanced-typography.adoc

```diff
@@ -0,0 +1,60 @@
+= Advanced Typography plugin
+:navtitle: Advanced Typography
+:description: Automatically or interactively replace typewriter-centric characters with characters preferred by typesetters.
+:description_short: Auomatically or interactively typeset {productname} documents.
+:keywords: plugin, typography, typesetting, smart quotes, dashes, em-dash, en-dash, quotation marks
+:pluginname: Advanced Typography
+:plugincode: typography
+:plugincategory: premium
+
+include::partial$misc/admon-premium-plugin.adoc[]
+
+include::partial$misc/admon-requires-6.3v.adoc[]
+
+The {pluginname} plugin, depending on its setting, automatically or interactively replaces common typewriter-style characters with their professional typesetter’s equivalents.
```

Comment:
I think that _interactively_ here suggests that you can approve / reject each one as it finds it, which is not the case. Is there a better choice of word here?

Reply:
Replaced introductory paragraph with equivalent paragraph (primarily the work of @kemister85) from 6.3 Release Notes section on Advanced Typography. This, among other things, deals with the problem of ‘interactive’ and ‘automatic’ being parsed unhelpfully by readers, as per @ephox-morgan’s note.